set.mm

```diff
@@ -127214,6 +127214,24 @@ Rational numbers (as a subset of complex numbers)
     LVOVHVRQVKVHVOVRVHVDVEVRVFBUCRUDUEVPVQADVKVDVEVQAUFZVDVKBSFZVEVSBUKVKASFVTV
     EVSAULABUGUHUIRUJUMUNUOUPUQURUSUTVFCDMVA $.
 
+  ${
+    $d A x y z $.
+    $( A positive rational is the quotient of two positive integers.
+       (Contributed by AV, 29-Dec-2022.) $)
+    elpq $p |- ( ( A e. QQ /\ 0 < A )
+                 -> E. x e. NN E. y e. NN A = ( x / y ) ) $=
```

Comment:
Maybe prove instead the biconditional
```
 elpq $p |- ( A e. QQ -> ( 0 < A <-> E. ... ) ) $=
```
?
Maybe move to Main ?

Reply:
I can (and will) prove 
```
elpqb $p |- ( ( A e. QQ /\ 0 < A ) <-> E. x e. NN E. y e. NN A = ( x / y ) ) $=
```

Reply:
You're right, that's better (prove it in place of elpq).

Reply:
Done